python/tester/upload_from_repo.py

```diff
@@ -142,15 +149,18 @@ def main():
         pool.join()
 
 
-def get_visit_list(butler, n_sample, **kwargs):
-    """Return a list of randomly selected raw visits in the butler repo.
+def get_visit_list(butler, n_sample, ordered=False, **kwargs):
+    """Return a list of selected raw visits in the butler repo.
 
     Parameters
     ----------
     butler : `lsst.daf.butler.Butler`
         The Butler in which to search for records of raw data.
-    n_sample: `int`
+    n_sample : `int`
         The number of visits to select.
+    ordered : `bool`
+        If `True`, return the first `n_sample` visits in the order of the visit
+        IDs. Otherwise, return `n_sample` randomly selected visit IDs.
```

Comment:
It would be difficult to generally support uploads with the actual relative timing. It depends on the input dataset selection.  For one, that mode doesn't make sense for the HSC-RC2 dataset which includes hand-picked data across 1.5 year. 

Leaving that out of scope for now. We might support some timing config later.

Reply:
Maybe... this is better named as "ordered mode" rather than "replay mode" which suggests more than it does.  Opinions?

Reply:
I like "ordered".